spa/src/hooks/useSentry.ts

```diff
@@ -0,0 +1,39 @@
+import { useEffect } from 'react';
+
+// Sentry
+import * as Sentry from '@sentry/react';
+import { BrowserTracing } from '@sentry/tracing';
+import { createBrowserHistory } from 'history';
+import { Route } from 'react-router-dom';
+
+// Const
+import { ENVIRONMENT, SENTRY_DSN_FRONTEND, SENTRY_ENABLE_FRONTEND } from 'settings';
+
+const history = createBrowserHistory();
+
+/**
+ * Typically, just loading sentry at build time is sufficient.
```

Comment:
```suggestion
 * Typically, just loading sentry at load time is sufficient.
```
I think that's what you mean here?